lib/livebook_web/live/session_live.ex

```diff
@@ -550,11 +550,13 @@ defmodule LivebookWeb.SessionLive do
 
   @impl true
   def handle_params(%{"cell_id" => cell_id}, _url, socket) do
+    assert_policy!(socket, :read)
```

Comment:
We don't strictly need the live action check, but it's safer to only handle_params in cases we actually expect them

```suggestion
  def handle_params(%{"cell_id" => cell_id}, _url, socket)
      when socket.assigns.live_action in [:cell_settings, :cell_upload] do
```